airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json

```diff
@@ -48,15 +48,22 @@
           }
         }
       },
-      "customer_id": {
-        "title": "Customer Id",
-        "type": "string",
-        "description": "Customer id must be specified as a 10-digit number without dashes. More instruction on how to find this value in our <a href=\"https://docs.airbyte.io/integrations/sources/google-adwords#setup-guide\">docs</a>"
+      "customer_ids": {
```

Comment:
It seems that we globally do not change the logic much. Can we preserve backward compatibility for previous versions, not rename the field for the specification, but simply expand its description, after all, now we already indicate a list of `customer_id` separated by commas, let users continue to enter `customer_id` separated by a comma in field `customer_id`, and programmatically we will create a list of customer IDs. You can update `title` for this field if you want, but let`s save native name. WDYT?

Reply:
Yes that sounds sensible, thank you very much for the review. I will adapt the points accordingly.